aikido_perception/include/aikido/perception/ConfigDataLoader.hpp

```diff
@@ -0,0 +1,84 @@
+#ifndef AIKIDO_PERCEPTION_CONFIG_DATA_LOADER_H
+#define AIKIDO_PERCEPTION_CONFIG_DATA_LOADER_H
+
+#include "yaml-cpp/yaml.h"
+#include <aikido/perception/yaml_conversion.hpp>
+#include <aikido/util/CatkinResourceRetriever.hpp>
+#include <dart/common/Console.h>
+#include <dart/common/LocalResourceRetriever.h>
+#include <Eigen/Geometry>
+#include <stdexcept>
+
+namespace aikido{
+namespace perception{
+
+class ConfigDataLoader
```

Comment:
:+1: Please add Doxygen docstrings to this class and its member functions (including all arguments).